examples/nlp/token_classification.py

```diff
@@ -0,0 +1,183 @@
+parser.add_argument("--save_step_freq", default=-1, type=int,
+                    help="Frequency of saving checkpoint \
+                    '-1' - step checkpoint won't be saved")
+parser.add_argument("--loss_step_freq", default=250, type=int,
+                    help="Frequency of printing loss")
+
+args = parser.parse_args()
+
+if not os.path.exists(args.data_dir):
+    raise FileNotFoundError("Dataset not found. For NER, CoNLL-2003 dataset"
+                            "can be obtained at"
+                            "https://github.com/kyzhouhzau/BERT"
+                            "-NER/tree/master/data.")
+
+nf = nemo.core.NeuralModuleFactory(backend=nemo.core.Backend.PyTorch,
+                                   local_rank=args.local_rank,
+                                   optimization_level=args.amp_opt_level,
+                                   log_dir=args.work_dir,
+                                   create_tb_writer=True,
+                                   files_to_copy=[__file__],
+                                   add_time_to_log_dir=True)
+
+nf.logger.info(args)
+
+output_file = f'{nf.work_dir}/output.txt'
+
+if args.bert_checkpoint is None:
+    """ Use this if you're using a standard BERT model.
+    To see the list of pretrained models, call:
+    nemo_nlp.huggingface.BERT.list_pretrained_models()
+    """
+    tokenizer = NemoBertTokenizer(args.pretrained_bert_model)
+    bert_model = nemo_nlp.huggingface.BERT(
+        pretrained_model_name=args.pretrained_bert_model)
+else:
+    """ Use this if you're using a BERT model that you pre-trained yourself.
+    Replace BERT-STEP-00.pt with the path to your checkpoint.
+    """
+    tokenizer = SentencePieceTokenizer(model_path=args.tokenizer_model)
+    tokenizer.add_special_tokens(["[MASK]", "[CLS]", "[SEP]"])
+
+    bert_model = nemo_nlp.huggingface.BERT(config_filename=args.bert_config)
+    bert_model.restore_from(args.bert_checkpoint)
+
+hidden_size = bert_model.local_parameters["hidden_size"]
+classifier = nemo_nlp.TokenClassifier(hidden_size=hidden_size,
+                                      num_classes=args.num_classes,
+                                      dropout=args.fc_dropout)
+punct_loss = nemo_nlp.TokenClassificationLoss(num_classes=args.num_classes)
+
+
+def create_pipeline(num_samples=-1,
+                    pad_label=args.none_label,
+                    max_seq_length=args.max_seq_length,
+                    batch_size=args.batch_size,
+                    local_rank=args.local_rank,
+                    num_gpus=args.num_gpus,
+                    mode='train',
+                    ignore_extra_tokens=args.ignore_extra_tokens,
+                    ignore_start_end=args.ignore_start_end,
+                    use_cache=args.use_cache):
+
+    nf.logger.info(f"Loading {mode} data...")
+    shuffle = args.shuffle_data if mode == 'train' else False
+
+    text_file = f'{args.data_dir}/text_{mode}.txt'
+    label_file = f'{args.data_dir}/labels_{mode}.txt'
```

Comment:
check to make sure these files exist. warn users if they don't.

Reply:
done